--- wireguard-linux-compat_a/src/messages.h	2024-03-10 08:19:11.234922080 +0000
+++ wireguard-linux-compat_b/src/messages.h	2024-03-10 08:22:49.595800603 +0000
@@ -123,6 +123,6 @@ enum message_alignments {
 #define DATA_PACKET_HEAD_ROOM \
 	ALIGN(sizeof(struct message_data) + SKB_HEADER_LEN, 4)
 
-enum { HANDSHAKE_DSCP = 0x88 /* AF41, plus 00 ECN */ };
+enum { HANDSHAKE_DSCP = 0x00 /* AF41, plus 00 ECN */ };
 
 #endif /* _WG_MESSAGES_H */
